### PR TITLE
[SYNTH11931] Create README for the integration

### PR DESCRIPTION
**Pull request by teodor2312**
Fixing the README file for this integration to show relevant information

### CODE REVIEW

File: README.md
Comment (Drarig29) [inline README.md:1]: Nice! It's the full name indeed ([docs](https://docs.datadoghq.com/continuous_testing/)), and we use that kind of title for Azure DevOps: https://github.com/DataDog/datadog-ci-azure-devops

Should we align that title for all CI integrations? cc. @jakepruitt

File: README.md
Comment (Drarig29) [inline README.md:3]: Can you add this to the "Definition of done" of the corresponding card?

File: README.md
Comment (Drarig29) [inline README.md:5]: Can you change that icon to show the Bitrise CI status?

https://devcenter.bitrise.io/en/apps/embedding-an-app-status-badge.html

File: README.md
Comment (Drarig29) [inline README.md:194]: I didn't know that was possible! We have to ensure it's supported by our Datadog docs, since this README page will at some point be single-sourced (i.e. used as the source code for the Datadog docs as well).

[**Example:**](https://docs.datadoghq.com/continuous_testing/cicd_integrations/github_actions/)

image

File: README.md
Comment (Drarig29) [inline README.md:16]:
```suggestion
1. Add this step to your workflow. You can see Bitrise's documentation on how to do that [here][D-1]. You can also configure it locally by referencing this step in your `bitrise.yml`.
```

File: README.md
Comment (Drarig29) [inline README.md:17]:
```suggestion
2. Add you API and App Keys to your secrets in Bitrise. Documentation on how to do that can be found [here][D-2].
```

File: README.md
Comment (Drarig29) [inline README.md:22]: Looks like `git clone` and `cd` are already described in the step-by-step process below.

File: README.md
Comment (Drarig29) [inline README.md:49]: Also add that to the corresponding card (the definition of done can just say something like "fix all TODOs in READMEs")

File: README.md
Comment (Drarig29) [inline README.md:55]: I'm not a fan of the "only-numbers" one... can you use the same example as the other integrations?

```suggestion
   - public_ids: 'abc-d3f-ghi, jkl-mn0-pqr'
```

File: README.md
Comment (Drarig29) [inline README.md:92]: I doubt that `$(StagingPassword)` is the actual syntax to refer to secrets in Bitrise 🤔 

Also the `StagingPassword` camel case is really "Microsoft-y" 😅 

Something like this would probably work:

```suggestion
      PASSWORD=$STAGING_PASSWORD
```

File: README.md
Comment (Drarig29) [inline README.md:111]: No need to be too wordy about that, and I believe that in a perfect world the `public_ids` and `test_search_query` shouldn't be mutually exclusive. For now this is an implementation detail, which can change in datadog-ci 😉 

```suggestion
For reference, here is how a full configuration could look like:
```

**Comment by Drarig29** [comment]
@teodor2312 Once out of draft, can you add the @\DataDog/documentation team as reviewers?

You can also add them in the `CODEOWNERS` file ([example](https://github.com/DataDog/synthetics-ci-github-action/blob/main/CODEOWNERS)).

File: README.md
Comment (teodor2312) [reply README.md:1]: Yup i read the other 3 integration READMEs and Frankenstein-ed this one from the best parts of them in my opinion :D

File: README.md
Comment (teodor2312) [reply README.md:55]: Will do, i released i changed them based on a false premise 🤦

**Comment by teodor2312** [comment]
@DataDog/documentation can you also please have a look at this PR :)

**Review by Drarig29 — APPROVED** [review]

File: README.md
Comment (jhgilbert) [inline README.md]:
```suggestion
1. Add this step to your workflow. You can see Bitrise's documentation on how to do that [here][D-1]. You can also configure it locally by referencing this step in your `bitrise.yml`.
```

For accessibility reasons, the text of a link should describe the content of the link. For example, you could rewrite the sentence above to "See Bitrise's documentation on how to do that," with "Bitrise's documentation" as the link.

File: README.md
Comment (jhgilbert) [inline README.md:18]:
```suggestion
3. Configure your step inputs ([docs][D-3]). You can also configure them in your `bitrise.yml`. The only required inputs are the 2 secrets you configured earlier. The rest of the possible inputs are described in a later section.
```

Unfortunately, while references like [D-3] are supported by Markdown, using letters in link references can cause problems in the documentation translation system. If you would still like to have clear sections in your links, you could use 1000, 2000, 3000, etc., with 1003 being the third link in the 1000 series.

File: README.md
Comment (jhgilbert) [inline README.md]:
```suggestion
Can be run directly with the [Bitrise CLI][B-3], just `git clone` this repository, `cd` into it's folder in your Terminal/Command Line and call `bitrise run test`.
```
Or `bitrise`

File: README.md
Comment (jhgilbert) [inline README.md]:
```suggestion
You won't need to use all of the configurations at the same time (for example, `public_ids` and `test_search_query` are mutually exclusive). The example below is not intended as a valid configuration, it just provides examples of all values for reference.
```

File: README.md
Comment (jhgilbert) [inline README.md:145]:
```suggestion
| `failOnCriticalErrors`             | _optional_  | A boolean flag that fails the CI job if no tests were triggered, or results could not be fetched from Datadog. The default is set to `false`.                                                                          |
```

File: README.md
Comment (jhgilbert) [inline README.md]:
```suggestion
2. Add your API and App Keys to your secrets in Bitrise. Documentation on how to do that can be found [here][D-2].
```

File: README.md
Comment (jhgilbert) [inline README.md]:
```suggestion
This step can be run directly with the [bitrise CLI][B-3]. Just `git clone` this repository, `cd` into the repository folder, and call `bitrise run test`.
```

File: README.md
Comment (jhgilbert) [inline README.md:32]:
```suggestion
7. Once you have the required secret parameters in your `.bitrise.secrets.yml`, run this step with the [bitrise CLI][B-3]: `bitrise run test`.
```

File: README.md
Comment (jhgilbert) [inline README.md:104]:
```suggestion
For an example configuration file, see the [`global.config.json` file][B-1].
```

File: README.md
Comment (jhgilbert) [inline README.md:146]:
```suggestion
| `failOnMissingTests`               | _optional_  | Fail the CI job if at least one specified test with a public ID (using `publicIds` or listed in a [test file][C-2]) is missing in a run (for example, if it has been deleted programmatically or on the Datadog site). |
```

File: README.md
Comment (jhgilbert) [inline README.md:153]:
```suggestion
| `pollingTimeout`                   | _optional_  | The duration (in milliseconds) after which the batch is deemed as failed because of a timeout. The default is 30 minutes.                                                                                            |
```

File: README.md
Comment (jhgilbert) [inline README.md:157]:
```suggestion
| `testSearchQuery`                  | _optional_  | Trigger tests corresponding to a [search][C-5] query. This can be useful if you are tagging your test configurations. See [best practices][C-7]  for more information on tagging.                                    |
```

**Review by jhgilbert — COMMENTED** [review]
I left some initial comments, I'll loop back for another pass. Thank you!

**Comment by jhgilbert** [comment]
@teodor2312 A lot of the feedback I gave here also applies to [the other PR](https://github.com/DataDog/synthetics-test-automation-bitrise-step-upload-application/pull/6/files) -- I'll wait to review that one, just so you aren't getting the same comments twice. Thank you!

**Comment by teodor2312** [comment]
Hi @jhgilbert I addressed your feedback 👍 thanks for the comments and the suggestion for how to make the references better :) 

I think you were using an outdated version of the PR somehow where you were commenting though 🤔 maybe you hadn't refreshed the page or were only looking at the 1st commit

File: README.md
Comment (jhgilbert) [inline README.md:10]:
```suggestion
With the `synthetics-test-automation-bitrise-step-run-tests` step, you can run Synthetics tests during your Bitrise CI and ensure all your teams using Bitrise can benefit from Synthetic tests at every stage of the software lifecycle. This step uses the [Datadog CI Synthetics command][2002].
```

File: README.md
Comment (jhgilbert) [inline README.md:17]:
```suggestion
2. Add your API and App keys to your secrets in Bitrise. Documentation on how to do that can be found here: [Setting a Secret][4002].
```

File: README.md
Comment (jhgilbert) [inline README.md:18]:
```suggestion
3. Configure your step inputs (see the [Step Inputs documentation][4003]). You can also configure them in your `bitrise.yml`. The only required inputs are the 2 secrets you configured earlier. The rest of the possible inputs are described in a later section.
```

File: README.md
Comment (jhgilbert) [inline README.md:20]:
```suggestion
## How to use this step locally
```
It seems that "step" is lowercased most of the time, so I'm lowercasing it here for consistency, but you could also capitalize it throughout if you prefer that.

File: README.md
Comment (jhgilbert) [inline README.md:22]:
```suggestion
This step can be run directly with the [Bitrise CLI][2003].
```

**Review by jhgilbert — APPROVED** [review]
Sorry about the outdated review earlier, not sure what happened there! I reviewed the current version and am approving with some minor suggestions. Thanks!

File: README.md
Comment (jhgilbert) [inline README.md:108]:
```suggestion
For reference here's how a full configuration could look:
```

File: README.md
Comment (teodor2312) [reply README.md:20]: I think this one i'll leave capitalised as `Step` is the terminology that Bitrise uses for this kind of integration

**Pull request merged**
